jetty-core/jetty-client/src/main/java/org/eclipse/jetty/client/transport/HttpReceiver.java

```diff
@@ -623,6 +637,12 @@ public long getLength()
             return source.getLength();
         }
 
+        @Override
+        public InvocationType getInvocationType()
+        {
+            return Invocable.getInvocationType(source);
```

Comment:
source is final, so perhaps work this out in the constructor?

Reply:
We can't make it a constant because `Content.Source` delegates the `InvocationType` to the demand callback that may change across invocations of `demand(Runnable)` -- although admittedly rare.